### ISSUE TITLE
Link to triggering commit

### ISSUE THREAD

**Issue by lpugin** [music-encoding/guidelines#37]
It would be useful to have in the dev version of the guidelines a link to the music-encoding/music-encoding repo that triggered the corresponding build. Maybe we can have this in the footer?

**Comment by ahankinson** [comment]
You mean in the markdown itself? It's in the commit message, but I just noticed that doesn't work.

**Comment by lpugin** [comment]
Yes, so we can see if from the web-page. This will also help to check/know that the built was successful.

**Comment by lpugin** [comment]
Can write is a plain text to the `_include/${MEI_VERSION}/build.txt` file? It can be for all version

**Comment by lpugin** [comment]
See https://github.com/music-encoding/guidelines/commit/ab56cc6ce2ffd4b8ad7330e97161ddaa382708cb
